ui/app/abilities/variable.js

```diff
@@ -2,6 +2,7 @@
 import { computed, get } from '@ember/object';
 import { or } from '@ember/object/computed';
 import AbstractAbility from './abstract';
+import { default as doesMatchPattern } from 'nomad-ui/utils/match-glob';
```

Comment:
`nit` `non-blocking`: This `import` statement looks funky. You should be able to just write:

```
import doesMatchPattern from 'nomad-ui/utils/match-glob';
```